tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/MsTestV2RetriesTests.cs

```diff
@@ -28,7 +28,7 @@ public MsTestV2RetriesTests(ITestOutputHelper output)
     protected override string TrueAtThirdRetry => "Samples.MSTestTestsRetries.TestSuite.TrueAtThirdRetry";
 
     [SkippableTheory]
-    [MemberData(nameof(PackageVersions.MSTest), MemberType = typeof(PackageVersions))]
+    [MemberData(nameof(PackageVersions.MSTest2Retries), MemberType = typeof(PackageVersions))]
```

Comment:
@tonyredondo Is this right? Wasn't sure as it was using `MSTest` (I think they map to the same versions though)